src/Data/Inject/Coproduct.purs

```diff
@@ -0,0 +1,23 @@
+module Data.Inject.Either where
```

Comment:
Should be `Data.Inject.Coproduct`?

Reply:
Uh yes :wink:

Reply:
Yeah, this change is required to compile - it collides with the other Data.Inject.Either module in this project.